datatypes/src/main/java/org/hyperledger/besu/datatypes/Hash.java

```diff
@@ -118,4 +118,19 @@ public static Hash fromHexString(final String str) {
   public static Hash fromHexStringLenient(final String str) {
     return new Hash(Bytes32.fromHexStringLenient(str));
   }
+
+  /***
+   * For logging purposes, this method returns a shortened hex representation
+   *
+   * @param hexRepresentation the string hex representation
+   * @return shortened string with only the beginning and the end of the hex representation
+   */
+  public static String shortenHexString(final String hexRepresentation) {
```

Comment:
Suggest renaming to `toShortString()` for consistency with the other method, and making it an instance method that formats itself and not a static method, so it is used like:
```
final Hash h = ....
final String s = h.toShortString()
```

Reply:
I was hesitating with this format, but yes, it makes sens